utils.go

```diff
@@ -365,3 +365,25 @@ func runProcess(container libcontainer.Container, config *specs.Process, listenF
 	}
 	return handler.forward(process)
 }
+
+func validateProcessSpec(spec ...interface{}) error {
```

Comment:
I think we can just use `func validateProcessSpec(spec specs.Process)` here because all we have to validate is `specs.Process` now, and if we need to validate more in specs, we can add back validateSpec like:
```
func validateSpec(spec *specs.Spec) error {
    if err := validateProcessSpec(spec.Process); err != nil {
        return err
    }
    <do-more-validate>
}
```